driver-tests/shading/shaded/pom.xml

```diff
@@ -53,6 +57,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
I've filed JAVA-3122 to add a test for ensuring dropwizard is correctly shaded (outside of OSGi)